### PR TITLE
Port new docs for System.CodeDom, System.Globalization and System.Runtime.CompilerServices, System.Net (EII) and System.Data.SqlTypes (EII)

### PR DESCRIPTION
**Pull request by carlossanlop**
Ports some missing docs from https://github.com/dotnet/runtime/issues/88561

### CODE REVIEW

**Review by wfurt — APPROVED** [review]
networking looks good to me. I wish we named parameters better than `s` Burt it seems like this is what we API approved.

**Comment by tarekgh** [comment]
Globalization looks good. it is really about the new numeric style `AllowBinarySpecifier`. CC @tannergooding if want to have a look.

**Review by buyaa-n — APPROVED** [review]
[System.CodeDom.Compiler/IndentedTextWriter.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/SomePorts80/xml/System.CodeDom.Compiler/IndentedTextWriter.xml) LGTM.

File: xml/System.Runtime.CompilerServices/InlineArrayAttribute.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Indicates that the instance's storage is sequentially replicated "length" times.</summary>
```
Comment (gewarren) [inline xml/System.Runtime.CompilerServices/InlineArrayAttribute.xml:27]:
```suggestion
    <summary>Indicates that the instance's storage is sequentially replicated <see cref="P:System.Runtime.CompilerServices.InlineArrayAttribute.Length" /> times.</summary>
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**